diploma-client/config/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');

```

Comment:
Seems like there is already a webpack-config section present in the vue.config.js